corehq/apps/registration/views.py

```diff
@@ -89,7 +89,7 @@ def register_user(request):
 
         context.update({
             'form': form,
-            'legacy_password': getattr(settings, 'ENABLE_DRACONIAN_SECURITY_FEATURES', False),
+            'show_password_feedback': not getattr(settings, 'ENABLE_DRACONIAN_SECURITY_FEATURES', False),
```

Comment:
i'm pretty sure it's correct. if `ENABLE_DRACONIAN_SECURITY_FEATURES` then we have stricter password requirements which are different from the front end library. context: https://github.com/dimagi/commcare-hq/pull/10864#issuecomment-199340457